recipes/flask-turbolinks/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flask-turbolinks" %}
+{% set version = "0.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9086fa5d425db3530efe7eb96d90067f0531158864f0e7130846e534069d6381" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.python.org/packages/4d/f3/e7b4ff1fa5c121e81480bf54dcfb8ce61ccab1153cc6d3a00c36f1e7d2a6/Flask-Turbolinks-0.2.0.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
```

Comment:
Is `flask` required to build the recipe?

Reply:
Is a `python setup.py build` without the `install_requires = ['Flask']` in `setup.py` enough to check this?

Reply:
Basically because of `--single-version-externally-managed`, you don't need anything in `install_requires` unless it gets imported directly or indirectly in the `setup.py`.

Reply:
Ah!! Very nice, then it should not be needed. I will remove it in the next commit.